services/worker/src/worker/config.py

```diff
@@ -288,13 +288,15 @@ def from_env(cls) -> "DuckDbIndexConfig":
 
 
 DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY = None
+DESCRIPTIVE_STATISTICS_TARGET_REVISION = "refs/convert/parquet"
```

Comment:
we already have PARQUET_AND_INFO_TARGET_REVISION, maybe reuse this?

Otherwise, if we create a new environment variable, we have to add it to the README, to the docker-compose files, and to the Helm chart (yes, it's a lot)